### ISSUE TITLE
no active internet connection - mac os install - virtual machine manager

### ISSUE THREAD

**Issue by peterandrewd** [foxlet/macOS-Simple-KVM#247]
I have tried ALL the different network options in virt manager settings. I have tried using a seperate wifi adaptor even.
I cannot even install macOS because it requires an internet connection to install, and the VM/guest doesn't pick up any WiFi adaptor or connection despite it being activated in settings

**Comment by peterandrewd** [comment]
Well? I’m pretty frustrated and not going to wait all day...

**Comment by foxlet** [comment]
I would recommend using `basic.sh` in the Terminal rather than virt-manager if you need networking out of the box. However, without any information on your distro or setup, there is no way to help.

**Comment by peterandrewd** [comment]
> I would recommend using `basic.sh` in the Terminal rather than virt-manager if you need networking out of the box. However, without any information on your distro or setup, there is no way to help.

Yeah I used basic.sh also and I have network connection with basic.sh but not on virt manager.
Btw I’m running Linux Mint 19.3

**Comment by peterandrewd** [comment]
I just want internet connectivity and USB Passthrough for iPhone.

**Comment by pablopenna** [comment]
+1. I have connectivity with basic.sh but not with virt-manager. I have tried changing the device type under NAT but still does not work.

**Comment by ArnaudValensi** [comment]
I have the same issue on manjaro/kde. Work with basic.sh but not with virt-manager. I tryed NAT and bridge.
I will post here if I figure out why.

**Comment by ArnaudValensi** [comment]
I solved the problem by using a bridged interface and adding `<model type="vmxnet3"/>`.
See https://old.reddit.com/r/VFIO/comments/lwe7x8/hackintosh_qemukvm_network_issues/
and https://github.com/foxlet/macOS-Simple-KVM/issues/456

**Comment by roelfdiedericks** [comment]
```
<interface type="user">
  <mac address="52:54:00:b5:1b:54"/>
  <model type="vmxnet3"/>
  <alias name="net0"/>
  <address type="pci" domain="0x0000" bus="0x00" slot="0x10" function="0x0"/>
</interface>

```
This seems to do the trick in virt-manager. The bus=0x00 seems to be the key
